test/test_owners.csv

```diff
@@ -1,535 +1,738 @@
+Federation replicasets Federated ReplicaSet should be deleted from underlying clusters when OrphanDependents is false,nikhiljindal,0,federation
+Federation replicasets Federated ReplicaSet should create and update matching replicasets in underling clusters,childsb,1,federation
+Federation replicasets Federated ReplicaSet should not be deleted from underlying clusters when OrphanDependents is nil,nikhiljindal,0,federation
+Federation replicasets Federated ReplicaSet should not be deleted from underlying clusters when OrphanDependents is true,nikhiljindal,0,federation
+Federation replicasets ReplicaSet objects should be created and deleted successfully,apelisse,1,federation
+Federation secrets Secret objects should be created and deleted successfully,pmorie,1,federation
+Federation secrets Secret objects should be deleted from underlying clusters when OrphanDependents is false,nikhiljindal,0,federation
+Federation secrets Secret objects should not be deleted from underlying clusters when OrphanDependents is nil,nikhiljindal,0,federation
+Federation secrets Secret objects should not be deleted from underlying clusters when OrphanDependents is true,nikhiljindal,0,federation
+Firewall rule should create valid firewall rules for LoadBalancer type service,rkouj,0,network
+Firewall rule should have correct firewall rules for e2e cluster,rkouj,0,network
+GCP Volumes GlusterFS should be mountable,nikhiljindal,0,storage
+GCP Volumes NFSv4 should be mountable for NFSv4,nikhiljindal,0,storage
+GKE local SSD should write and read from node local SSD,fabioy,0,storage
+GKE node pools should create a cluster with multiple node pools,fabioy,1,cluster-lifecycle
+Garbage Collection Test: * Should eventually garbage collect containers when we exceed the number of dead containers per container,Random-Liu,0,cluster-lifecycle
+Garbage collector should delete RS created by deployment when not orphaning,rkouj,0,cluster-lifecycle
+Garbage collector should delete pods created by rc when not orphaning,justinsb,1,cluster-lifecycle
+Garbage collector should orphan RS created by deployment when deleteOptions.OrphanDependents is true,rkouj,0,cluster-lifecycle
+Garbage collector should orphan pods created by rc if delete options say so,fabioy,1,cluster-lifecycle
+Garbage collector should orphan pods created by rc if deleteOptions.OrphanDependents is nil,zmerlynn,1,cluster-lifecycle
+"Generated release_1_5 clientset should create pods, delete pods, watch pods",rrati,0,api-machinery
+"Generated release_1_5 clientset should create v2alpha1 cronJobs, delete cronJobs, watch cronJobs",soltysh,1,api-machinery
+HA-master survive addition/removal replicas different zones,derekwaynecarr,0,api-machinery
+HA-master survive addition/removal replicas multizone workers,rkouj,0,api-machinery
+HA-master survive addition/removal replicas same zone,derekwaynecarr,0,api-machinery
+AppArmor should enforce an AppArmor profile,derekwaynecarr,0,node
```

Comment:
@calebamiles Looks like the file got repeated here. Could it be an issue with the [`hack/update_owners.py`](https://github.com/kubernetes/kubernetes/blob/master/hack/update_owners.py) script?